src/components/SelectionList/BaseListItem.tsx

```diff
@@ -34,6 +34,8 @@ function BaseListItem<TItem extends ListItem>({
     const StyleUtils = useStyleUtils();
     const {hovered, bind} = useHover();
 
+    const isSelectable = item.isSelectable === undefined || item.isSelectable;
```

Comment:
Because we have to keep possibility to click on the item to go to the edit page and disable only checkbox (what was not possible before)

Reply:
@filip-solecki What do you think about using `isDisabledCheckbox` instead of `isSelectable`? I think this approach makes the condition simple and safer.